config/initializers/pagy.rb

```diff
@@ -1,5 +1,6 @@
 require "pagy/extras/trim"
 require "pagy/extras/countless"
+require "pagy/extras/size"
```

Comment:
Let's keep this for now.

I'm creating an issue to use integer sizes in favor of using the array legacy API.